ros2run/ros2run/command/run.py

```diff
@@ -26,6 +28,11 @@ class RunCommand(CommandExtension):
 
     def add_arguments(self, parser, cli_name):
         arg = parser.add_argument(
+            '--prefix',
+            help="Prefix command, which should go before the executable. "
+                 "Command must be wrapped in quotes if it contains spaces"
```

Comment:
This needs a space before the closing quote (to have spacing before the opening parenthesis).